<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="recvB">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>recv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStartTimer">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>startTimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStartTimer_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>stopTimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>create file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readFile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>read file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="readChunck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>read next  chunk</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>32</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>chunk  Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearText">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>clear text</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bytesReadEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>bytes Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>fileSize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileSizeEite">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>453</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
